docs/install/installEverest.md

```diff
@@ -46,10 +61,24 @@ To install and provision Percona Everest to Kubernetes:
         everestctl install --namespaces dev,prod --operator.mongodb=true --operator.postgresql=true --operator.xtradb-cluster=true --skip-wizard
         ```
 
-    !!! warning "Important"
-        Ensure to copy the authorization token displayed on the terminal in this step. You will need this token to log in to the Percona Everest UI.    
+3.  When you install Percona Everest, an `admin` user is automatically created. To retrieve the password for the `admin` user, run the following command:
+
+    ```sh
+    everestctl accounts initial-admin-password
+    ```
+
+    We strongly recommend that you update the `admin` password after installation.
```

Comment:
Since our recommendation is to update the password (step 4.) I would remove this as a step and maybe add a note saying you can retrieve the automatically generated password by running `everestctl accounts initial-admin-password` while highlighting that this automatically generated password isn't stored securely.

Reply:
Done.